### PR TITLE
Pde hessian0

### PR DESCRIPTION
**Pull request by TTitscher**
This pull request basically adds a Hessian0() to cell and integrand. As lively debated in #43, this method can be later replaced by Integrate(Operator) or something. It returns a `NuTo::DofMatrix` and we can start to write an assembler and eventually solve the first global system.

The other commits are pure renaming, replacing the `rStuff` pattern in the arguments with `stuff`.

### CODE REVIEW

File: src/mechanics/cell/Cell.h [hunk 1]
```diff
@@ -41,6 +41,24 @@ class Cell : public CellInterface
         return gradient;
     }
 
+    //! @brief builds the hessian0 matrix
+    DofMatrix<double> Hessian0() override
+    {
+        DofMatrix<double> hessian0;
```
Comment (Psirus) [inline src/mechanics/cell/Cell.h:47]: Will this not simply call the default constructor of Eigen, possibly resulting in a nonzero initial value?

**Review by Psirus — APPROVED** [review]

File: src/mechanics/cell/Cell.h (hunk 1 above)
Comment (TTitscher) [reply src/mechanics/cell/Cell.h:47]: This is kind of a special behavior of the `DofContainer` things. They allow - as I called it - [uninitialized addition](https://github.com/nutofem/nuto/blob/master/test/mechanics/nodes/DofMatrix.cpp#L38). So this line will create a DofMatrix<double> containing zero sized vectors that is resized when needed.

Comment (Psirus) [reply src/mechanics/cell/Cell.h:47]: Okay. Not the most intuitive behaviour though.

Comment (TTitscher) [reply src/mechanics/cell/Cell.h:47]: If you can come up with a better solution, please let me know. The problem is that the cell does not know the size of this container, or how many dof types it contains. We could do sth like
~~~c++
DofMatrix<double> hessian0 = mIntegrands[0].Gradient(...);
for (i = 1, ...) 
   hessian0 += mIntegrands[i].Gradient(....);
~~~
this seemed kind of silly though.

**Pull request merged**